definition-src/Game/LambdaHack/Content/PlaceKind.hs

```diff
@@ -58,6 +58,11 @@ data Fence =
   | FNone   -- ^ skip the fence and fill all with the place proper
   deriving (Show, Eq)
 
+-- Places are rooms and other dungeon features, their names can be seen on a level map by aiming at a position that is an entry to the place 
```

Comment:
To make it a haddock (visible when you do `cabal haddock` and view the html), change the start of this line to `-- | Place`.